### DISCUSSION TITLE
How to distinguish between packages with the same name but different paths in autocomplete？

### DISCUSSION THREAD

**Discussion by shizhx** [golang/vscode-go#2471] [Help]
![image](https://user-images.githubusercontent.com/7520007/191405723-7a2a3dd3-e64f-4c03-9d42-f6d1a5bcbe1b.png)

**Comment by hyangah** [comment]
Looks like the language server is not working. 

When gopls is working correctly, the editor shows 

![Screen Shot 2022-09-26 at 9 59 59 AM](https://user-images.githubusercontent.com/4999471/192295998-364a4d61-a30b-4b76-be62-d67473fee679.png)

I also enabled semantic tokens too.

**Reply by shizhx** [reply]
Language server is working correctly, but when my project depends on multiple similar packages(multiple `context.Context` in different packages), I cannot distinguish from them.
